### PR TITLE
[4.0] Option colour in menu settings

### PR DESCRIPTION
**Pull request by brianteeman**
There is a class on all option fields for a component that adds the colour green to positive settings. However this class was not present for the same options when creating a menu item

### Testing Instructions
Create any menu item and on the options tab check to see that the value for Show or Yes are green when selected

### CODE REVIEW

**Comment by 1apweb** [comment]
I have tested this item :white_check_mark: successfully on 70d6fda281ae14b0b24785d50b15fbca25df3c64<br /><br />It's fine now<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26735">issues.joomla.org/tracker/joomla-cms/26735</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 59c5fcfc7119f2e7eaaccc2c7abe4499fe99b983<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26735">issues.joomla.org/tracker/joomla-cms/26735</a>.</sub>

**Comment by richard67** [comment]
@1apweb Could you repeat your test with the latest version of this PR? There have been made some updates meanwhile. Thanks in advance.

File: components/com_content/tmpl/archive/default.xml [hunk 1]
```diff
@@ -125,7 +125,7 @@
 				label="COM_CONTENT_FIELD_INFOBLOCK_POSITION_LABEL"
 				default=""
 				useglobal="true"
-				class="chzn-color"
+				class="custom-select-color-state"
```
Comment (Quy) [inline components/com_content/tmpl/archive/default.xml:128]: Should this be colored?

**Comment by 1apweb** [comment]
I have tested this item :white_check_mark: successfully on 59c5fcfc7119f2e7eaaccc2c7abe4499fe99b983<br /><br />the colors are displayed well<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26735">issues.joomla.org/tracker/joomla-cms/26735</a>.</sub>

File: components/com_content/tmpl/archive/default.xml (hunk 1 above)
Comment (brianteeman) [reply components/com_content/tmpl/archive/default.xml:128]: on the basis that it was in j3 then yes

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26735">issues.joomla.org/tracker/joomla-cms/26735</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**